### PR TITLE
Fix WL-2704

### PR DESCRIPTION
**Pull request by ghost**
Removes the default value for the choice of turnitin repository. This is also configured by the server option `turnitin.repository.setting.value` which will override this to set a default. We need to make sure this is not set by the server.

### CODE REVIEW

File: assignment-tool/tool/src/java/org/sakaiproject/assignment/tool/AssignmentAction.java [hunk 1]
```diff
@@ -1814,7 +1814,7 @@ protected void setAssignmentFormContext(SessionState state, Context context)
 		context.put("value_AllowStudentView", state.getAttribute(NEW_ASSIGNMENT_ALLOW_STUDENT_VIEW) == null ? Boolean.toString(ServerConfigurationService.getBoolean("turnitin.allowStudentView.default", false)) : state.getAttribute(NEW_ASSIGNMENT_ALLOW_STUDENT_VIEW));	
 	
 		List<String> subOptions = getSubmissionRepositoryOptions();
-		String submitRadio = ServerConfigurationService.getString("turnitin.repository.setting.value",null) == null ? NEW_ASSIGNMENT_REVIEW_SERVICE_SUBMIT_NONE : ServerConfigurationService.getString("turnitin.repository.setting.value");
+		String submitRadio = ServerConfigurationService.getString("turnitin.repository.setting.value",null) == null ? "" : ServerConfigurationService.getString("turnitin.repository.setting.value");
```
Comment (buckett) [inline assignment-tool/tool/src/java/org/sakaiproject/assignment/tool/AssignmentAction.java]: I think you can just pass empty string as the default, eg ServerConfigurationService.getString("turnitin.repository.setting.value","")

**Comment by ghost** [comment]
Good point. Done.

File: assignment-tool/tool/src/java/org/sakaiproject/assignment/tool/AssignmentAction.java [hunk 2]
```diff
@@ -4600,8 +4600,13 @@ else if (sameAssignmentTitleInContext(assignmentRef, title, (String) state.getAt
 		
 		//set submit options
 		r = params.getString(NEW_ASSIGNMENT_REVIEW_SERVICE_SUBMIT_RADIO);
-		if(r == null || (!NEW_ASSIGNMENT_REVIEW_SERVICE_SUBMIT_STANDARD.equals(r) && !NEW_ASSIGNMENT_REVIEW_SERVICE_SUBMIT_INSITUTION.equals(r)))
-			r = NEW_ASSIGNMENT_REVIEW_SERVICE_SUBMIT_NONE;
+		if (r == null && Boolean.TRUE.toString().equals(state.getAttribute(NEW_ASSIGNMENT_USE_REVIEW_SERVICE))) {
```
Comment (buckett) [inline assignment-tool/tool/src/java/org/sakaiproject/assignment/tool/AssignmentAction.java:4603]: You can do Boolean.parseBoolean(state.getAttribute(NEW_ASSIGNMENT_USE_REVIEW_SERVICE)), which I would use if the rest of the code wasn't using this pattern already.

**Comment by baholladay** [comment]
This commit broke the VeriCite content review implementation by requiring VeriCite to select a Turnitin repository

**Comment by buckett** [comment]
This commit is in the local Oxford deployment of Sakai. The one that went into Sakai is in: https://github.com/steveswinsburg/sakai/pull/5/files

**Comment by baholladay** [comment]
Sorry for posting, I was confused with the pull requests links. I added the comment on the Sakai branch

**Pull request merged**